From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Marijn Suijten <marijns95@gmail.com>
Date: Thu, 11 Feb 2021 22:01:54 +0100
Subject: [PATCH] cirrus_sony: Use tabs instead of spaces

---
 hal/audio_extn/cirrus_sony.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hal/audio_extn/cirrus_sony.c b/hal/audio_extn/cirrus_sony.c
index cd07ed5a5181a220fb011c332e121b7a590f0c47..1904eac03f6c1e7f1457a529eab67dce7fc1a2e1 100644
--- a/hal/audio_extn/cirrus_sony.c
+++ b/hal/audio_extn/cirrus_sony.c
@@ -843,10 +843,10 @@ retry_fw:
         ret = 0;
     } else {
         /*
-	 * Since we are using a poor hack to load the firmware, we cannot know
+         * Since we are using a poor hack to load the firmware, we cannot know
          * if the firmware was found nor if it finished loading remotely.
-	 * We also don't know how much time does the chip require to actually
-	 * boot it, so we will sleep and retry for X times, until it loads and
+         * We also don't know how much time does the chip require to actually
+         * boot it, so we will sleep and retry for X times, until it loads and
          * boots, or we assume that something went wrong: in that case the
          * only thing left to do is to return an error, hoping that developers
          * will catch it before going crazy...
