From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Frajo Haider <f_haider@gmx.at>
Date: Wed, 17 Mar 2021 12:10:05 +0200
Subject: [PATCH] cirrus_sony: Fix mono calibration firmware names

---
 hal/audio_extn/cirrus_sony.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/hal/audio_extn/cirrus_sony.c b/hal/audio_extn/cirrus_sony.c
index 3996a08f84b711b87eed17b1901a76ed1bd257bb..3f6595c18d1a50646ddaeefc701aaf4ca5e5165d 100644
--- a/hal/audio_extn/cirrus_sony.c
+++ b/hal/audio_extn/cirrus_sony.c
@@ -1234,15 +1234,13 @@ static int cirrus_do_fw_mono_download(int do_reset) {
         goto exit;
     }
 
-    ret = cirrus_set_mixer_array_by_name(CIRRUS_CTL_CALI_CAL_STATUS,
-                                         &handle.spkr.status, 4);
+    ret = cirrus_write_cal_status(&handle.spkr, 0);
     if (ret < 0) {
         ALOGE("%s: Cannot set calibration status", __func__);
         goto exit;
     }
 
-    ret = cirrus_set_mixer_array_by_name(CIRRUS_CTL_CALI_CAL_CHECKSUM,
-                                         &handle.spkr.checksum, 4);
+    ret = cirrus_write_cal_checksum(&handle.spkr, 0);
     if (ret < 0) {
         ALOGE("%s: Cannot set calibration checksum", __func__);
         goto exit;
