From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Frajo Haider <f_haider@gmx.at>
Date: Wed, 17 Mar 2021 12:09:04 +0200
Subject: [PATCH] cirrus_sony: Retry loading the fw in mono calibration case.

---
 hal/audio_extn/cirrus_sony.c | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/hal/audio_extn/cirrus_sony.c b/hal/audio_extn/cirrus_sony.c
index 1edbfcf95a95c4d07973b1c84a1d879dc910142a..f953b63dc49d218d6a3935ad43940eb00181bf00 100644
--- a/hal/audio_extn/cirrus_sony.c
+++ b/hal/audio_extn/cirrus_sony.c
@@ -1248,9 +1248,14 @@ exit:
 
 static int cirrus_do_fw_mono_download(int do_reset) {
     bool cal_valid = false, status_ok = false, checksum_ok = false;
-    int ret = 0;
+    int i, max_retries = 24, ret = 0;
 
-    ret = cirrus_exec_fw_download("Protection", 0, do_reset);
+    for (i = 0; i < max_retries; i++) {
+        ret = cirrus_exec_fw_download("Protection", 0, do_reset);
+        if (ret == 0)
+            break;
+        usleep(500000);
+    }
     if (ret != 0) {
         ALOGE("%s: Cannot send Protection firmware: bailing out.",
               __func__);
